packages/table/src/Table.story.js

```diff
@@ -700,37 +700,71 @@ storiesOf("Table/SelectionBar", module)
     ))
   )
   .add(
-    "Primary button",
+    "One action button",
     withInfo()(() => (
       <SelectionBar
         selectedRows={[{ name: "foo" }]}
         clearSelection={action("clear selection called")}
-        primaryAction={action("primary button clicked")}
-        primaryButtonLabel="Primary Action"
-      />
+      >
+        {selectedRows => (
```

Comment:
this looks odd to me, is this a common pattern used by other libs? why not a property instead of a child?

Reply:
We use the same concept on `Popover` component. The only diff is that there we pass a property called `content`

Reply:
Hmmm... Yep, as we have access to the selectedRows outside the `SelectionBar` I agree with you... It makes sense to render children normally

Reply:
@vNakamura just tested with regular children and it works fine... Let's change that?!